$OpenBSD: patch-test-framework-quickcheck2_cabal,v 1.3 2016/01/27 20:22:09 kili Exp $

Builds fine with QuickCheck-2.8. Another example of hackage
inconsistency (upper bound in the distfile is still 2.8, but the
webinterface shows 2.9).

--- test-framework-quickcheck2.cabal.orig	Fri Apr 11 12:23:46 2014
+++ test-framework-quickcheck2.cabal	Wed Jan 27 21:06:58 2016
@@ -25,7 +25,7 @@ Library
         Exposed-Modules:        Test.Framework.Providers.QuickCheck2
 
         Build-Depends:          test-framework >= 0.7.1,
-                                QuickCheck >= 2.4 && < 2.8,
+                                QuickCheck >= 2.4 && < 2.9,
                                 extensible-exceptions >= 0.1.1 && < 0.2.0
         if flag(base3)
                 Build-Depends:          base >= 3 && < 4, random >= 1
